Bixal/move.mil
# Feature 156725284 display current page title breadcrumb

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- includes a preprocessor that gets the current path
- new styles to match live site

## Testing

To verify the changes proposed in this pull request…

1. pull code locallly
2. cd /move-mil/web/themes/custom/move_mil
3. make build
4. make cr
5. navigate to any page on the site to see breadcrumbs for that page that shows path as visited and current page as inactive.

## Screenshots

![screen shot 2018-04-18 at 4 56 53 pm](https://user-images.githubusercontent.com/37077057/38957816-85fa9fe8-4329-11e8-9423-dbeea201d944.png)
